src/backend/distributed/commands/utility_hook.c

```diff
@@ -237,8 +244,23 @@ multi_ProcessUtility(PlannedStmt *pstmt,
 		return;
 	}
 
-	ProcessUtilityInternal(pstmt, queryString, context,
-						   params, queryEnv, dest, completionTag);
+	UtilityHookLevel++;
+
+	PG_TRY();
+	{
+		bool executedDDLJob = ProcessUtilityInternal(pstmt, queryString, context,
```

Comment:
maybe call this `executedOnCitusTable` or such to make it a little more clear

Reply:
removed that variable, so resolving